content/en/synthetics/mobile_app_testing/mobile_app_tests/restricted_networks.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Run Mobile App tests from Restricted Networks
+description: "Run Mobile App tests from restricted networks"
+further_reading:
+- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
+  tag: "Blog"
+  text: "Best practices for creating end-to-end tests"
+- link: "/synthetics/mobile_app_testing/mobile_app_tests"
+  tag: "Documentation"
+  text: "Learn how to create Synthetic mobile app tests"
+- link: "/synthetics/mobile_app_testing/settings"
+  tag: "Documentation"
+  text: "Learn how to upload your iOS or Android mobile applications"
+cascade:
+  algolia:
+    tags: ['mobile_testing']
+---
+
+## Overview
+
+Some of your applications may not be available to the public internet because they are either accessing development or local environments, or they are internal applications intended for users within your corporate network, for example, your corporate intranet or VPN. 
```

Comment:
```suggestion
Some of your applications might not be available to the public Internet because they are accessing development or local environments, or they are internal applications intended for users within your corporate network (for example, your corporate intranet or VPN). 
```